FreshRSS/FreshRSS
# fix #3661: not logged in user: button to mark all as read leads to HTTP403 error

Closes #3661

Changes proposed in this pull request:
- if user is not logged in (=default/guest user) than no button to mark all articles as read. Show only a text


How to test the feature manually:
1. with default/guest user not logged in
2. scroll to the end of articles list
3. see: "There are no more articles" instead of "mark all as read" button
![grafik](https://user-images.githubusercontent.com/1645099/136671405-d08a3452-1632-4e1c-86de-5ffa6dfb25ff.png)


Pull request checklist:
